packages/hardhat-core/src/internal/core/providers/accounts.ts

```diff
@@ -235,6 +251,7 @@ abstract class SenderProvider extends ProviderWrapper {
       method === "eth_call" ||
       method === "eth_estimateGas"
     ) {
+      // TODO: Should we validate this type?
```

Comment:
I didn't validate some the wrappers that don't have local implementations of the methods. These just add some missing params and fwd the request.